DOCS.md

```diff
@@ -172,12 +172,13 @@ This posts weather information of a given location.
 
 __Usage__
 
-`@weather <zipcode | location name>`
+`@weather <zipcode | location name> > <degree type>`
 
 __Arguments__
 
 * `zipcode`: A five digit US zipcode.
 * `location name`: A letter and space only location name, or city name and state separated by a comma and a space.
+* `degree type`: A degree type to display the weather, the parameter can be 'C' for Celsius or 'F' for Fahrenheit. The default type is Fahrenheit.
```

Comment:
Yeah I was thinking about that, I'm going to do some toUpperCase stuff and it should be good to go.